authentication/http_recordings_test.go

```diff
@@ -194,3 +216,14 @@ func redactDomain(i *cassette.Interaction, domain string) {
 	i.Response.Body = strings.ReplaceAll(i.Response.Body, domainParts[0], recordingsDomain)
 	i.Request.Body = strings.ReplaceAll(i.Request.Body, domainParts[0], recordingsDomain)
 }
+
+func verifyClientAssertion(t *testing.T, clientAssertion string) {
+	keyS := jwtPublicKey
+
+	key, err := jwk.ParseKey([]byte(keyS), jwk.WithPEM(true))
```

Comment:
```suggestion
	keys := jwtPublicKey

	key, err := jwk.ParseKey([]byte(keys), jwk.WithPEM(true))
```

Reply:
The reassigning here is a leftover before I extracted to a common key so I'll fix this to just remove `keyS`

Reply:
Fixed in 4f1ea30418bb3789e920307da5e65652520252f1